dev-scheme/guile/guile-2.2.3.recipe

```diff
@@ -0,0 +1,109 @@
+SUMMARY="GNU Ubiquitous Intelligent Language for Extensions"
+DESCRIPTION="Guile is a library designed to help programmers create flexible \
+applications. Using Guile in an application allows programmers to write \
+plug-ins, or modules (there are many names, but the concept is essentially \
+the same) and users to use them to have an application fit their needs."
+HOMEPAGE="https://www.gnu.org/software/guile/"
+COPYRIGHT="1993-2018 Aubrey Jaffer, George Carrette, et al."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/guile/guile-2.2.3.tar.gz"
+CHECKSUM_SHA256="87ee07caef33c97ddc74bf3c29ce7628cfac12061f573e4a29a3a1176754610a"
+PATCHES="guile-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libguileVersion="1.3.0"
+libguileVersionCompat="$libguileVersion compat >= ${libguileVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+
```

Comment:
Suggestion:
```diff
 portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+portVers="${portVersion%.*}"
```
Well, we don't really need this, but it can be handy since we can later replace each occurence of `2.2` by `$portVers`.